src/app/client/src/app/modules/workspace/components/content-editors/new-collection-editor/new-collection-editor.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="!showLoader && editorConfig">
-  <lib-editor-base *ngIf="!showQuestionEditor" [editorConfig]="editorConfig" (editorEmitter)="editorEventListener($event)"></lib-editor-base>
+  <lib-editor *ngIf="!showQuestionEditor" [editorConfig]="editorConfig" (editorEmitter)="editorEventListener($event)"></lib-editor>
```

Comment:
@vaibhavbhuva Why we required two lines? Both are expecting same  input as `editorConfig` & output as `editorEventListener`. remove other one.

Reply:
@pallakartheekreddy Fixed.